pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1144,6 +1144,9 @@ self: super: {
       })
       super.persistent-postgresql;
 
+  # Test suite requires a later version of persistent-test which depends on persistent 2.14
```

This PR LGTM.

Maybe also link to the discussion on stackage:

```suggestion
  # Test suite requires a later version of persistent-test which depends on persistent 2.14
  # https://github.com/commercialhaskell/stackage/issues/6884
```